examples/with-prisma/packages/database/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "database",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "clean": "rimraf .turbo node_modules dist",
+    "dev": "tsup --watch",
```

Comment:
We should move as many of these to `turbo`'s pipeline as possible

Reply:
I've moved the majority of the `database` package `prisma` scripts to the Turbo pipeline, including the `generate` script to generate typedefs.

However, one I am unsure of is the migration dev commend (`prisma migrate dev`) as in my testing Turbo will proceed straight through the user input prompt to name the migration & create a new migration with no name.
Do we want that as the default @jaredpalmer ?

Reply:
No, let's keep that out of `turbo` for now